file_store/src/verified_mapping_event.rs

```diff
@@ -0,0 +1,63 @@
+use crate::{
+    traits::{MsgDecode, MsgTimestamp, TimestampDecode, TimestampEncode},
+    Error, Result,
+};
+use chrono::{DateTime, Utc};
+use helium_proto::services::poc_mobile::VerifiedSubscriberMappingEventV1;
+use serde::{Deserialize, Serialize};
+use sqlx::Row;
+
+#[derive(Clone, Deserialize, Serialize, Debug, PartialEq)]
+pub struct VerifiedSubscriberMappingEvent {
+    pub subscriber_id: Vec<u8>,
+    pub total_reward_points: u64,
+    pub timestamp: DateTime<Utc>,
+}
+
+impl MsgDecode for VerifiedSubscriberMappingEvent {
```

Comment:
I dont see any filestore support for the ingest report ( which is a wrapper around the ingestor received request ) ie this msg in my original proto branch
https://github.com/helium/proto/blob/andymck/subscriber-mapping-support/src/service/poc_mobile.proto#L228

Reply:
Will add